frontend/ui/data/css_tweaks.lua

```diff
@@ -478,6 +478,26 @@ sup { font-size: 50% !important; vertical-align: super !important; }
 sub { font-size: 50% !important; vertical-align: sub !important; }
                 ]],
             },
+            (function()
+                local sub_table = {
+                    title = _("Override font-based normal line leight"),
```

Comment:
```suggestion
                    title = _("Override font-based normal line height"),
```
Wondering if "normal" should be quoted (as it targets those having `line-height: normal` - or if all this is just normal enough for leaving it unquoted :) ? @Frenzie ?